en/SUMMARY.md

```diff
@@ -13,8 +13,8 @@
     * [Windows](setup/dev_env_windows.md)
     * [Fast RTPS installation](setup/fast-rtps-installation.md)
   * [Building the Code](setup/building_px4.md)
-  * [Writing your First Application](tutorials/tutorial_hello_sky.md)
-  * [Module Template](tutorials/module_template.md)
+  * [Writing your First Application](apps/hello_sky.md)
+  * [Application/Module Template](apps/module_template.md)
```

Comment:
FYI the term module is quite generic. I put Application here to make it (even more) clear that this is a "next step".